### DISCUSSION TITLE
failed to install guppy screen - SSL problem, curl failed.

### DISCUSSION THREAD

**Discussion by z226** [Guilouz/Creality-Helper-Script-Wiki#594] [Issues]
I tried to install the guppy screen and got this error.  I have no clue what to do

Are you sure you want to install Guppy Screen ? (y/n): y

 Do you want to install Nightly Build or Release Build ? (nightly/release): nightly

Info: Downloading Guppy Screen...

curl: (60) SSL certificate problem: certificate is not yet valid
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

**Comment by z226** [comment]
it worked today

**Comment by moysesth** [comment]
Just to share I had the same issue. The problem was the date and time on the printer config.
I've adjust it manually running `date -s "2024-07-19 10:00:00"`, after it worked fine.

**Comment by gusarg81** [comment]
Hi, is failing to me too today. I just removed Guppyscreen to install again because suddenly was not connecting with the printer. When I wasted to install again, I got:

curl: invalid option -- 's'

EDIT: Figured it out: the problem was when I´ve enabled force_login in moonraker.
